test/test_unit_strings.cpp

```diff
@@ -101,6 +101,56 @@ TEST(unitStrings, prefixes)
     EXPECT_EQ(to_string(precise::micro * precise::L), "uL");
 }
 
+TEST(unitStrings, si_prefixes)
+{
+    EXPECT_EQ(to_string(unit_from_string("mm")), "mm");
+    EXPECT_EQ(to_string(unit_from_string("cm")), "cm");
+    EXPECT_EQ(to_string(unit_from_string("um")), "um");
+    EXPECT_EQ(to_string(unit_from_string("nm")), "nm");
+    EXPECT_EQ(to_string(unit_from_string("pm")), "pm");
+    EXPECT_EQ(to_string(unit_from_string("fm")), "fm");
+    EXPECT_EQ(to_string(unit_from_string("am")), "am");
+    EXPECT_EQ(to_string(unit_from_string("zm")), "zm");
+    EXPECT_EQ(to_string(unit_from_string("ym")), "ym");
+    EXPECT_EQ(to_string(unit_from_string("dm")), "dm");
+
+    EXPECT_EQ(to_string(unit_from_string("km")), "km");
+    EXPECT_EQ(to_string(unit_from_string("Km")), "km");
```

Comment:
Any particular reason for supporting uppercase `K` here?

Reply:
Correct or not I have seen `Kg` get used in a number of contexts for kilogram.  I suppose I could just remove the support for general upper case 'K' here and specifically support `Kg`